### PR TITLE
Add functions to interfaces for FederatedAttestations and Escrow

### LINKED ISSUE #9549: [ASv2] Add functions to the FederatedAttestations interface

### PR DESCRIPTION
**Pull request by isabellewei**
### Description

Filled out interface files

### Related issues

- Fixes #9549

### CODE REVIEW

File: packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol [hunk 1]
```diff
@@ -13,4 +35,19 @@ interface IFederatedAttestations {
...
+    bytes32 identifier,
+    address issuer,
+    address account,
+    address signer,
+    uint64 issuedOn,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  ) external view;
 }
```
Comment (isabellewei) [inline packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol:60]: not sure why, but it seems like `pure` functions are not included in interfaces

Comment (eelanagaraj) [reply packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol:60]: also not sure, though there is one in `ICeloVersionedContract.sol`, maybe it makes sense for us to include the getAttestationHash function?

**Review by eelanagaraj — COMMENTED** [review]
nice! other than `IEscrow.sol` though, it looks like most other interfaces don't include the parameter names, can you remove those from the `IFederatedAttestations` function sigs? Up to you whether you want to do that for `IEscrow` (+ maybe even fix the old sigs so that it's consistent)

File: packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol (hunk 1 above)
Comment (isabellewei) [reply packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol:60]: sure, no harm in adding :)

File: packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+  ) external;
+  function revokeAttestation(bytes32, address, address) external;
+  function batchRevokeAttestations(
+    address issuer,
+    bytes32[] calldata identifiers,
+    address[] calldata accounts
+  ) external;
+
+  // view functions
   function lookupAttestations(bytes32, address[] calldata)
```
Comment (eelanagaraj) [inline packages/protocol/contracts/identity/interfaces/IFederatedAttestations.sol]: depending on whether we go with named parameters or not, let's make all the functions in the file consistent; reminder to possibly add param names to the lookup functions

**Pull request merged**